src/platform-integration/web/wasm.md

```diff
@@ -1,53 +1,62 @@
 ---
 title: Support for WebAssembly (Wasm)
-description:
+description: >-
   Current status of Flutter's experimental support for WebAssembly (Wasm).
 short-title: Wasm
-last-update: May 23, 2023
+last-update: October 26, 2023
 ---
 
 The Flutter and Dart teams are excited to add
 [WebAssembly](https://webassembly.org/) as a compilation target when building
 applications for the web.
 
-{{site.alert.warning}} Development of WebAssembly support for Dart and Flutter
-remains ongoing.
-
-You can preview Wasm and WebAssembly garbage collection (WasmGC) in the
-[`master` channel](https://github.com/flutter/flutter/wiki/flutter-build-release-channels#master).
-As Flutter provides these features as previews,
-ongoing development might result in frequent changes.
-Revisit this page for the latest updates.
-
-**_Last updated {{page.last-update}}_**
-
+{{site.alert.warning}}
+  Development of WebAssembly support for Dart and Flutter remains ongoing.
+  
+  You can preview Wasm and WebAssembly garbage collection (WasmGC)
+  on the [`master` channel][].
+  As Flutter provides these features as previews,
+  ongoing development might result in frequent changes.
+  Revisit this page for the latest updates.
+  
+  **_Last updated {{page.last-update}}_**
 {{site.alert.end}}
 
+[`master` channel]: https://github.com/flutter/flutter/wiki/flutter-build-release-channels#master
+
 ### Background
 
-To compile Dart and Flutter to Wasm, you need a browser that supports
-[WasmGC](https://github.com/WebAssembly/gc/tree/main/proposals/gc). The Wasm
-standard plans to add WasmGC to help garbage-collecting languages like Dart
-execute code in an efficient manner.
+To compile Dart and Flutter to Wasm, you need
```

Comment:
whitespace churn makes it hard to review! 😄

Reply:
Ah sorry, I updated to closer match our standards. I'll split it into different commits next time!

The main change in the PR is mentioning Chrome 119 and Firefox 120 without flags, and being more explicit about updating to the latest `main`.